_includes/modern-appendix.md

```diff
@@ -506,10 +506,12 @@ Sent to a client to let them know who set the topic (`<nick>`) and when they set
 
 ### `RPL_INVITING (341)`
 
-      "<client> <channel> <nick>"
+      "<client> <nick> <channel>"
 
 Sent as a reply to the [`INVITE`](#invite-message) command to indicate that the attempt was successful and the client with the nickname `<nick>` has been invited to `<channel>`.
 
+NB: The last two parameters are in the opposite order from the RFC, they have indeed been switched in [an erratum](https://www.rfc-editor.org/errata/eid2821).
```

Comment:
Let's get rid of this line. For the purposes of Modern we ignore that the RFCs exist.

Reply:
Done!